packages/gatsby-plugin-treat/tests/styling.test.js

```diff
@@ -0,0 +1,99 @@
+import execa from 'execa';
+import * as path from 'path';
+import waitForLocalhost from 'wait-for-localhost';
+import getStyles from '../../../test-helpers/getStyles';
+
+const gatsbyServerPort = 9999;
+const gatsbyBinaryPath = path.resolve(__dirname, '../../../node_modules/.bin/gatsby');
```

Comment:
Not a big issue but here's a little utility we use on other projects to solve this issue. 

```js
const resolveBin = (packageName, binName) => {
  const packageJson = require(`${packageName}/package.json`);
  const binPath =
    typeof packageJson.bin === 'string'
      ? packageJson.bin
      : packageJson.bin[binName || packageName];

  return require.resolve(path.join(packageName, binPath));
};
```

Reply:
I've just inlined the function in the latest commit but did you have anywhere in mind where these utilities would live?

Reply:
Never mind, found `test-helpers/`